libs/blocks/ost/ost.js

```diff
@@ -269,7 +275,8 @@ export default async function init(el) {
       },
       onReady: () => {
         if (!window.adobeIMS.isSignedInUser()) {
-          window.adobeIMS.signIn();
+          const { signInContext } = getConfig();
```

Comment:
Since OST (Offer Selector Tool) is only used by GWP and not available to the end user, I think we could keep it out of scope of this ticket
https://milo.adobe.com/tools/ost